pry-byebug.gemspec

```
@@ -22,5 +22,5 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.7.0"
 
   gem.add_runtime_dependency "byebug", "~> 11.0"
-  gem.add_runtime_dependency "pry", ">= 0.13", "< 0.15"
+  gem.add_runtime_dependency "pry", ">= 0.13", "< 0.16"
```

To avoid waiting 1 year in the future for new version to get released, I recommend

```suggestion
  gem.add_runtime_dependency "pry", ">= 0.13"
```

---

It's dangerous because of breaking changes in new versions.

---

this is development only gem, so it is not dangerous if it stops working. And highly unlikely that pry introduces breaking changes with each new release